### PR TITLE
📝 Fix suggested cURL interceptor on README

### PR DESCRIPTION
**Pull request by lohhans**
### New Pull Request Checklist

- [x] I have read the [Documentation](https://pub.dev/documentation/dio/latest/)
- [x] I have searched for a similar pull request in the [project](https://github.com/cfug/dio/pulls) and found none
- [x] I have updated this branch with the latest `main` branch to avoid conflicts (via merge from master or rebase)
- [x] I have added the required tests to prove the fix/feature I'm adding
- [x] I have updated the documentation (if necessary)
- [x] I have run the tests without failures
- [x] I have updated the `CHANGELOG.md` in the corresponding package

### Additional context and info (if any)

Changes the suggested interceptor to create cURL requests, the current one doesn't work on Dio 5.x.x versions, so this new package is functional for those versions. (Apparently the old package was abandoned by the developer, this other suggested one was born from a fork of the previous one).

### CODE REVIEW

**Comment by ueman** [comment]
Please also adjust the chinese readme. Thanks for the PR.

**Comment by lohhans** [comment]
@ueman Chinese readme updated! 😄

**Comment by ueman** [comment]
Thanks!

**Review by ueman — APPROVED** [review]

File: dio/README-ZH.md [hunk 1]
```diff
@@ -99,7 +99,7 @@ void getHttp() async {
...
 | [dio_smart_retry](https://github.com/rodion-m/dio_smart_retry)                                         | [![Pub](https://img.shields.io/pub/v/dio_smart_retry.svg)](https://pub.flutter-io.cn/packages/dio_smart_retry)                         | 支持灵活地请求重试                                          |
 | [http_certificate_pinning](https://github.com/diefferson/http_certificate_pinning)                     | [![Pub](https://img.shields.io/pub/v/http_certificate_pinning.svg)](https://pub.flutter-io.cn/packages/http_certificate_pinning)       | 用于 Flutter 的 HTTPS 证书锁定                            |
-| [curl_logger_dio_interceptor](https://github.com/OwnWeb/curl_logger_dio_interceptor)                   | [![Pub](https://img.shields.io/pub/v/curl_logger_dio_interceptor.svg)](https://pub.flutter-io.cn/packages/curl_logger_dio_interceptor) | 用于 Flutter 的 CURL 命令生成器                            |
+| [dio_intercept_to_curl](https://github.com/blackflamedigital/dio_intercept_to_curl)                   | [![Pub](https://img.shields.io/pub/v/dio_intercept_to_curl.svg)](https://pub.dev/packages/dio_intercept_to_curl) | 用于 Flutter 的 CURL 命令生成器                            |
```
Comment (ueman) [inline dio/README-ZH.md:102]:
```suggestion
| [dio_intercept_to_curl](https://github.com/blackflamedigital/dio_intercept_to_curl)                   | [![Pub](https://img.shields.io/pub/v/dio_intercept_to_curl.svg)](https://pub.flutter-io.cn/packages/dio_intercept_to_curl) | 用于 Flutter 的 CURL 命令生成器                            |
```

**Pull request merged**